### PR TITLE
Silent renew

### PR DESCRIPTION
**Pull request by shaunluttin**
https://github.com/openiddict/openiddict-samples/issues/11

### CODE REVIEW

File: samples/ImplicitFlow/AuthorizationServer/Startup.cs [hunk 1]
```diff
@@ -26,6 +26,9 @@ public class Startup {
                 .EnableLogoutEndpoint("/connect/logout")
                 .EnableIntrospectionEndpoint("/connect/introspect")
                 .EnableUserinfoEndpoint("/Account/Userinfo")
+                // Set absurdly low token lifetimes to demonstrate automatic token renewal
+                .SetIdentityTokenLifetime(System.TimeSpan.FromSeconds(30))
```
Comment (PinpointTownes) [inline samples/ImplicitFlow/AuthorizationServer/Startup.cs]: Add a `using System;` to get rid of the namespace :smile:

File: samples/ImplicitFlow/AureliaApp/package.json [hunk 2]
```diff
@@ -1,67 +1,46 @@
...
-  "version": "0.1.0",
-  "author": "Shaun Luttin <shaun@bigfont.ca>",
-  "license": "MIT",
+  "name": "aurelia-open-id-connect-demo",
+  "description": "A demo to shows aurelia-open-id-connect working with an OpenIddict.",
+  "version": "0.2.0",
   "repository": {
     "type": "git",
-    "url": "https://github.com/openiddict/openiddict-samples.git"
+    "url": "https://github.com/shaunluttin/aurelia-open-id-connect"
```
Comment (PinpointTownes) [inline samples/ImplicitFlow/AureliaApp/package.json]: Please revert.

File: samples/ImplicitFlow/AureliaApp/package.json [hunk 3]
```diff
@@ -1,67 +1,46 @@
 {
-  "name": "aurelia-openiddict",
-  "description": "A sample that shows aurelia-openid-client working with OpenIddict.",
-  "version": "0.1.0",
-  "author": "Shaun Luttin <shaun@bigfont.ca>",
-  "license": "MIT",
+  "name": "aurelia-open-id-connect-demo",
+  "description": "A demo to shows aurelia-open-id-connect working with an OpenIddict.",
```
Comment (PinpointTownes) [inline samples/ImplicitFlow/AureliaApp/package.json]: `working with an OpenIddict` sounds odd :sweat_smile:

**Comment by shaunluttin** [comment]
@PinpointTownes This PR requires further changes before merging. Those changes are under development in https://github.com/shaunluttin/openiddict-samples/tree/spa-sample-deploy-azure. Once those changes are done, I will merge them into this PR and let you know that it's ready for review and merge.

**Pull request closed without merging**